test/README.md

```diff
@@ -13,278 +13,268 @@
-| `team create`                                           | Y         | Y
-| `team delete`                                           | Y         | Y
-| `team describe`                                         | Y         | Y
-| `team list`                                             | Y         | Y
-| `user invite`                                           | N         | Y
-| `user delete`                                           | N         | Y
-| `user describe`                                         | Y         | Y
-| `performanceAdvisor namespace list`                     | Y         | Y
-| `performanceAdvisor slowQueryLogs list`                 | Y         | Y
-| `performanceAdvisor suggestedIndexes list`              | Y         | Y
-| `performanceAdvisor slowOT enable`                      | Y         | Y
-| `performanceAdvisor slowOT disable`                     | Y         | Y
-| `serverless create`                                     | Y         | Y
-| `serverless delete`                                     | Y         | Y
-| `serverless describe`                                   | Y         | Y
-| `serverless list`                                       | Y         | Y
-| `serverless watch`                                      | Y         | Y
-| `serverless update`                                     | Y         | Y
-| `serverless backup snapshots list`                      | Y         | Y
-| `serverless backup snapshots describe`                  | Y         | Y
-| `serverless backup snapshots watch`                     | Y         | Y
-| `livemigrations link create`                            | Y         | Y
-| `livemigrations link delete`                            | Y         | Y
-| `livemigrations validation create`                      |           | Y
-| `livemigrations create`                                 |           | Y
-| `livemigrations cutover`                                | N         | Y
-| `completion bash`                                       | Y         | Y
-| `completion zsh`                                        | Y         | Y
-| `completion fish`                                       | Y         | Y
-| `completion powershell`                                 | Y         | Y
-| `setup`                                                 | Y         | Y
-| `register`                                              | N         | Y
-| `kubernetes config generate`                            | Y         | Y
-| `kubernetes config apply`                               | Y         | Y
-| `kubernetes operator install`                           | Y         | Y
-| `federatedAuthentication federationSettings describe`   | Y         | Y
+|                        Command                        | E2E Atlas | Atlas |
+|:-----------------------------------------------------:|:---------:|:-----:|
+|                `alerts config create`                 |     Y     |   Y   |
```

Comment:
Did you intend to update this file as part of this PR?

Reply:
yes, intellij auto formatted the table to fix some missaligments